protos/sharding_rules.proto

```diff
@@ -22,7 +22,9 @@ message AddShardingRuleRequest {
 
 message AddShardingRuleReply {}
 
-message ListShardingRuleRequest {}
+message ListShardingRuleRequest {
```

Comment:
Do we have implementation for this? 
Also, we should support `LIST DATASPACE` in spqr-console

Reply:
Added